terracotta/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>terracotta-core-root</artifactId>
+        <groupId>org.terracotta.internal</groupId>
+        <version>5.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>terracotta</artifactId>
```

Comment:
As mentioned in the email thread, this module is used for below reasons: 

* terracotta package has required server class path in manifest and used in start scripts.
* used for copying dependencies

Could you add mvn "install" phase to cloudbees build job to get the build passing? Let me know if we need to create a issue for this.